Firebase/android/app/src/main/res/layout/merge_create_channel_view.xml

```diff
@@ -3,25 +3,46 @@
   android:layout_width="match_parent"
   android:layout_height="match_parent">
 
-  <include layout="@layout/toolbar_layout" />
-
-  <EditText
-    android:id="@+id/newChannelName"
-    android:layout_width="match_parent"
-    android:layout_height="wrap_content"
-    android:hint="@string/insert_channel_name" />
+  <include layout="@layout/toolbar_non_elelvated_layout" />
 
   <Switch
     android:id="@+id/privateChannelSwitch"
     android:layout_width="match_parent"
-    android:layout_height="wrap_content"
-    android:text="@string/private_channel" />
+    android:layout_height="48dp"
+    android:background="@color/background_grey_alt"
+    android:paddingLeft="16dp"
+    android:paddingRight="16dp"
+    android:text="@string/private_channel"
+    android:textColor="@color/text_grey"
+    android:textSize="14sp" />
```

Comment:
This is a first pass, I'll be extracting all styles when doing the review sitting down with @qqipp for design adjustments.